lib/manager/bundler/readme.md

```diff
@@ -46,78 +36,86 @@ No
 
 #### Can package files have "local" links to each other that need to be resolved?
 
+No
+
 #### Is there reason why package files need to be parsed together (in serial) instead of independently?
 
----
+No
 
 #### What format/syntax is the package file in? e.g. JSON, TOML, custom?
 
----
+Ruby syntax
 
 #### How do you suggest parsing the file? Using an off-the-shelf parser, using regex, or can it be custom-parsed line by line?
 
----
+I'm thinking about to use something that can transform ruby code to abstract syntax tree
 
 #### Does the package file structure distinguish between different "types" of dependencies? e.g. production dependencies, dev dependencies, etc?
 
-No
-
----
+Yes, but not sure that it's relevant to Renovate purposes
```

Comment:
It's not relevant to Renovate purposes, *but* users may want to add rules based on the group names. e.g. do things only for dev or production dependencies. I think we need to fit this into our "depType" model, but maybe make it an array.